test/extended/util/client.go

```diff
@@ -608,7 +626,10 @@ func (c *CLI) AdminBuildClient() buildv1client.Interface {
 }
 
 func (c *CLI) AdminConfigClient() configv1client.Interface {
```

Comment:
Good point. The right course of actions is to replace the `oc get` with the code equivalent. I don't think this can be easily detected without grepping/taking a look.

Reply:
fully replacing the logic of `oc get` with all the options is not a simple task and might get too complex. Can we opt out for something simpler like injecting forbidden groups into the CLI struct? And then just going over `commandArgs` and panic if we find these groups.

I think this might be sufficient since I am not seeing anyone using `oc` with `config.openshift.io` group. If somebody starts using it, we can either force them to use the go client or implement the oc mock logic later.

Reply:
> And then just going over commandArgs and panic if we find these groups.

Not every kind is going to be explicitly typed with the config.openshift.io group. In which case it's up to the KA to decide which group takes precedence.

Reply:
yes, in general it would get more complicated. I was looking for more of an temporary bandaid. Nevertheless this PR is pretty large already. Maybe the best would be to pursue this as a followup?

Reply:
This will require further analysis. Nevertheless, wherever possibly I prefer to replace `oc` CLI commands with their config client equivalents when accessing the config API. Let's make it on per case basis.